petab/v2/lint.py

```diff
@@ -290,7 +289,7 @@ def run(self, problem: Problem) -> ValidationIssue | None:
             except KeyError:
                 messages.append(
                     f"Observable {m.observable_id} used in measurement "
-                    f"table is not defined."
+                    f"table is not defined in the observable table."
```

Comment:
Separate suggestions due to github...

```suggestion
                    f"table but is not defined in the observable table."
```